### PR TITLE
1UPDATE

### PR DESCRIPTION
**Pull request by VinayVini2501**

### CODE REVIEW

File: src/test/java/CustomerModule.java [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+public class CustomerModule {
+
+	public static void main(String[] args)
+	{
+		System.out.println(" this is emty");
```
Comment (NagarajJkd) [inline src/test/java/CustomerModule.java:6]: add more printing statements

Comment (VinayVini2501) [reply src/test/java/CustomerModule.java:6]: OK I ILL WRITE

**Review by VinayVini2501 — COMMENTED** [review]
ok

File: src/test/java/com/myNewProject/UserModule.java [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
 	@Test
-	public void userLogin() {
+	public void userLogin()
+	{
+		System.out.println("name plz");
+		
 		
-		Reporter.log("User login is success");
-		Repoter.log("Launch the application");
-		Repoter.log("");
```
Comment (NagarajJkd) [inline src/test/java/com/myNewProject/UserModule.java:13]: accepted

File: src/test/java/com/myNewProject/UserModule.java [hunk 3]
```diff
@@ -1,15 +1,14 @@
 package com.myNewProject;
 
-import org.testng.Reporter;
 import org.testng.annotations.Test;
 
 public class UserModule {
 
 	@Test
-	public void userLogin() {
```
Comment (NagarajJkd) [inline src/test/java/com/myNewProject/UserModule.java:9]: change the class name

**Review by NagarajJkd — CHANGES_REQUESTED** [review]

**Review by NagarajJkd — APPROVED** [review]
approved

**Pull request merged**